fenix/app/src/main/java/org/mozilla/fenix/home/collections/CollectionMenuItem.kt

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.home.collections
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import mozilla.components.browser.state.selector.normalTabs
+import mozilla.components.feature.tab.collections.TabCollection
+import org.mozilla.fenix.R.string
+import org.mozilla.fenix.compose.MenuItem
+import org.mozilla.fenix.compose.inComposePreview
+import org.mozilla.fenix.ext.components
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Constructs and returns the default list of menu options for a [TabCollection].
+ *
+ * @param collection [TabCollection] for which the menu will be shown.
+ * Might serve as an argument for the callbacks for when the user interacts with certain menu options.
+ * @param onOpenTabsTapped Invoked when the user chooses to open the tabs from [collection].
+ * @param onRenameCollectionTapped Invoked when the user chooses to rename the [collection].
+ * @param onAddTabTapped Invoked when the user chooses to add tabs to [collection].
+ * @param onDeleteCollectionTapped Invoked when the user chooses to delete [collection].
+ */
+@Composable
+fun getMenuItems(
```

Comment:
Perhaps this should just live in https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/home/collections/CollectionViewHolder.kt where it is used and we can turn this into a private function.

Reply:
I would also suggest this, as it may allow us to not have to do the `if` with `LocalContext.current.components.core.store.state.normalTabs.isNotEmpty()` in `hasOpenTabs()`.